### DISCUSSION TITLE
Better translations

### DISCUSSION THREAD

**Discussion by lessborders** [directus/directus#15636] [Feature Requests]
The app doesnt actually support an intuitive way to manage content translations natively - you have to set up your own translations table and schemas - if anyone wants to make an extension to make native content translations with a better UI, that would be great!

Idea:
![image](https://user-images.githubusercontent.com/100006598/190926457-f3b9ac89-937b-4f7a-a64f-b6208ba053f0.png)

Field editing:
![image](https://user-images.githubusercontent.com/100006598/190926585-abc5abe4-44bc-4ead-975a-917176d5ec5f.png)

Field creation:
![image](https://user-images.githubusercontent.com/100006598/190926448-49b1a510-e899-4a0c-8206-d08c3eebd9f6.png)

**Comment by lessborders** [comment]
After playing around with the standard functionality a bit, there's actually some really cool stuff you can do for fallbacks translations. (even in JSON repeaters!)

```
const page = await directus.items('pages').readByQuery({
    fields: '*,translations.*',
    filter: id,
  })

const translations = page.translations?.find(
  (page: any) => page.languages_code == currentLocaleCode
)
const fallback = page.value.translations?.find(
  (page: any) => page.languages_code == defaultLocaleCode
)

page.title = translations?.title || fallback?.title
page.content = { ...fallback?.content, ...translations?.content } // this is a repeater field
```

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
